mappings/net/minecraft/block/Material.mapping

```diff
@@ -3,54 +3,76 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15933 PISTON Lnet/minecraft/class_3614;
 	FIELD field_15934 SNOW_BLOCK Lnet/minecraft/class_3614;
+		COMMENT Material for full sized snow blocks.
 	FIELD field_15935 PLANT Lnet/minecraft/class_3614;
-	FIELD field_15936 CLAY Lnet/minecraft/class_3614;
+		COMMENT Material for plants such as flowers and crops
+	FIELD field_15936 MOB_DERIVED Lnet/minecraft/class_3614;
+		COMMENT Material for blocks that are a product of mob interactions such as honey or slime, or the infested blocks. Includes clay but not bone blocks.
 	FIELD field_15937 CAKE Lnet/minecraft/class_3614;
 	FIELD field_15938 BAMBOO_SAPLING Lnet/minecraft/class_3614;
 	FIELD field_15939 blocksLight Z
 	FIELD field_15940 blocksMovement Z
-	FIELD field_15941 EARTH Lnet/minecraft/class_3614;
+	FIELD field_15941 TOPSOIL Lnet/minecraft/class_3614;
+		COMMENT Material for the top layer of soil. Path, dirt, podzol, soul soil, farmland and similar.
 	FIELD field_15942 GLASS Lnet/minecraft/class_3614;
+		COMMENT Material for glass and glass-like blocks (includes sea lanterns and conduits).
 	FIELD field_15943 FIRE Lnet/minecraft/class_3614;
 	FIELD field_15944 color Lnet/minecraft/class_3620;
-	FIELD field_15945 ORGANIC Lnet/minecraft/class_3614;
+	FIELD field_15945 SOLID_ORGANIC Lnet/minecraft/class_3614;
+		COMMENT Organic blocks that are solid, eg. hay, target, or grass blocks.
 	FIELD field_15946 BAMBOO Lnet/minecraft/class_3614;
 	FIELD field_15947 UNDERWATER_PLANT Lnet/minecraft/class_3614;
-	FIELD field_15948 SNOW Lnet/minecraft/class_3614;
-	FIELD field_15949 ANVIL Lnet/minecraft/class_3614;
+		COMMENT Material for underwater plants without the replaceable property.
+	FIELD field_15948 SNOW_LAYER Lnet/minecraft/class_3614;
+		COMMENT Material for non-full blocks of snow. Has the replaceable property.
+	FIELD field_15949 REPAIR_STATION Lnet/minecraft/class_3614;
+		COMMENT Material for blocks that can repair tools, ie. grindstone and anvils.
 	FIELD field_15950 replaceable Z
 	FIELD field_15951 burnable Z
 	FIELD field_15952 BARRIER Lnet/minecraft/class_3614;
 	FIELD field_15953 METAL Lnet/minecraft/class_3614;
-	FIELD field_15954 PUMPKIN Lnet/minecraft/class_3614;
+	FIELD field_15954 GOURD Lnet/minecraft/class_3614;
+		COMMENT Material for gourds. Includes the carved pumpkin and jack o' lantern.
 	FIELD field_15955 TNT Lnet/minecraft/class_3614;
 	FIELD field_15956 REPLACEABLE_PLANT Lnet/minecraft/class_3614;
 	FIELD field_15957 CARPET Lnet/minecraft/class_3614;
 	FIELD field_15958 ICE Lnet/minecraft/class_3614;
+		COMMENT Material for ice that can melt. See {@link #DENSE_ICE} for unmeltable ice.
 	FIELD field_15959 AIR Lnet/minecraft/class_3614;
 	FIELD field_15960 pistonBehavior Lnet/minecraft/class_3619;
 	FIELD field_17008 SHULKER_BOX Lnet/minecraft/class_3614;
 	FIELD field_22223 NETHER_WOOD Lnet/minecraft/class_3614;
+		COMMENT Material for blocks crafted from stems (Nether wood logs).
```

Comment:
The nether fungi aren't wood.

Reply:
Unfortunately, mojang considers them wood.
![image](https://user-images.githubusercontent.com/4873274/79671156-33103880-8196-11ea-8619-132e9efe2d34.png)

Reply:
Well, they're wood-like but not "nether wood logs".

Reply:
The tags call them wood, they belong to the logs category, they can be used to craft wood items such as chests and sticks - they are functionally wood.

Reply:
Just "nether logs" should suffice; logs are type of wood

Reply:
logs will not, as the planks and other nether wood blocks use this material.
![image](https://user-images.githubusercontent.com/4873274/79671929-de6fbc00-819b-11ea-8725-bdbb35df16df.png)

Reply:
`Material for blocks crafted from stems (Nether logs).`
what's wrong with this description